monitoring/loadtest/README.md

```diff
@@ -0,0 +1,35 @@
+# LoadTest tool
+
+The LoadTest tool is based on [Locust](https://docs.locust.io/en/stable/index.html) which provides a UI for controlling the number of Users to spawn and make random requests. Currently its configured to make the request in the ratio 10 x Create ISA : 5 x Update ISA : 100 x Get ISA : 1 x Delete ISA. This means the User is 10 times likely to Create an ISA vs Deleting an ISA, and 10 times more likely to Get ISA vs Creating an ISA and so on. Subscription workflow is heavier on the Write side with the ratio of 100 x Create Sub : 50 x Update Sub : 20 x Get Sub : 5 x Delete Sub.
+
+# Adjusting workload ratio
+In each files every action has a weight declared in the `@task(n)` decorator. You can adjust the value of `n` to suite your needs
+
+# Run Locally without Docker
+1. Go to the repository's root directory. We have to execute from root directory due to our directory structure choice.
+1. Create vitrual environment `virtualenv --python python3 ./monitoring/loadtest/env`
+1. Install dependencies `pip install -r ./monitoring/loadtest/requirements.txt`
+1. Configure your OAuth Server Endpoint in `./monitoring/loadtest/locust_files/conf.py`
+
+    a. Currently it only supports Dummy OAuth Server, if there is a request to support other Auth mechaisms, we can implement them.
```

Comment:
I think one of the immediately useful things to do with this tool will be to run it on real deployments, so I think it's likely worth supporting the general auth mechanism in monitorlib.  This should be pretty easy; see my comment below.

Reply:
What I will do what the prober tests does. Allow the user to use whatever class we have in infrastructure.auth